lib/resque/scheduler.rb

```diff
@@ -208,18 +210,18 @@ def enqueue_delayed_items_for_timestamp(timestamp)
         end
       end
 
+      def enqueue(config)
+        enqueue_from_config(config)
+      rescue => e
+        Resque::Scheduler::Plugin.run_failure_hooks(failure_handler, config, e)
```

Comment:
No strong opinions from me. 

Thoughts on saving the default behaviour (logging to `logger`) and in addition to that having a backend? Is that what resque/resque does? e.g.
```ruby
Resque::Scheduler.failure_handlers.each do |handler|
  handler.on_failure(config, e)
end
```